src/server/updater/kdcupdater.cpp

```diff
@@ -471,13 +492,27 @@ bool NSISUpdater::handleStartup() {
 
 void NSISUpdater::slotSetSeenVersion() {
     ParametersCache::instance()->parameters().setSeenVersion(updateInfo().version().toStdString());
+    setDownloadState(LastVersionSkipped);
     ExitCode exitCode = ParametersCache::instance()->save();
     if (exitCode != ExitCodeOk) {
         LOG_WARN(Log::instance()->getLogger(), "Error in ParametersCache::saveParameters");
         return;
     }
 }
 
+
+void NSISUpdater::slotUnsetSeenVersion() {
```

Comment:
I named it as such to match with void NSISUpdater::slotSetSeenVersion() {. This class has not been refactored since ownCloud, so there are many "weird" things like that. I tried to make the least amount of changes possible because this class cannot be thoroughly tested, and we will be implementing a brand new updater soon.